Samples/Chirper/ChirperGrains/ChirperAccount.cs

```diff
@@ -31,7 +31,6 @@ public class ChirperAccountState
         public string UserAlias { get; set; }
     }
 
-    [Reentrant]
```

Comment:
Since the 2 users try to follow each other in parallel is technically possible, I kept it reentrant.
Nevertheless I did find some potential issues with #1185, so I took the main idea an iterated on it. If you agree, I can make the change in the other piece of code too. It is a little more complex, but at the same time covers some edge cases, and since people may copy/paste from the sample, it might be good to have